estuary/utils/general.py

```diff
@@ -399,3 +434,39 @@ def set_story_labels(requested_node_label, results, reverse=False):
         idx += delta
         node_idx += delta
     return results
+
+
+def get_siblings_description(story_node_id, story_node_story_flow, backward):
+    """
+    Generate a description of the siblings.
+
+    :param string story_node_id: the id of the current story node
+    :param dict story_node_story_flow: has forward and backward relationships of current item
+    :param bool backward: determines the relationship direction the passed in node has with the
+    siblings in the story
+    :return: returns the appropriate siblings title
+    :rtype: string
+    """
+    def _get_plural_label(label):
+        if label.endswith('y'):
+            label = label[:-1] + 'ies'
+        else:
+            label = label + 's'
+        return label
+
+    if backward:
+        rel_direction = 'backward'
+    else:
+        rel_direction = 'forward'
+
+    label = story_node_story_flow['{0}_label_display'.format(rel_direction)]
```

Comment:
Rename this to `rel_label` because `label` sounds like it's the story node's label.